wptview.css

```diff
@@ -0,0 +1,73 @@
+:root {
+    font-family: "Fira Sans", "DejaVu Sans", "Bitstream Vera Sans", "Sans";
+    font-size: 12 px;
+    padding: 0;
+    margin:0;
+    background-color: #f8f8f8;
+}
+
+body {
+    margin: 0;
+}
+
+h1 {
```

Comment:
The interface looks a lot better than before, except this. Maybe a font change will help here?